src/components/MAPI/organizations/OrganizationDetailGeneral/utils.ts

```diff
@@ -147,43 +127,6 @@ function appendMasterStats(
   }
 }
 
-async function fetchNodePoolListForCluster(
```

Comment:
I moved all these to a common place, since they also came in handy with this implementation.